streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -729,6 +729,13 @@ class ZStream[-R, +E, +A](val process: ZManaged[R, E, Pull[R, E, A]]) extends Se
   final def drop(n: Int): ZStream[R, E, A] =
     self.zipWithIndex.filter(_._2 > n - 1).map(_._1)
 
+  /**
+   * Drops all elements of the stream until the specified predicate evaluates
+   * to `true`.
+   */
+  def dropUntil(pred: A => Boolean): ZStream[R, E, A] =
+    dropWhile(!pred(_)).drop(1)
```

Comment:
That's a really confusing implementation - it doesn't match the documentation.
"Until" in programming usually means not including when the predicate evaluates to `true`.
In fs2 this methods is called `dropThrough` and resp. `takeThrough`.